AI/kaggle_notebooks/patchtst/notebook.ipynb

```
@@ -0,0 +1,98 @@
+    "- Train period: 2015-01-01 ~ 2023-12-31\n",
+    "- Output: patchtst_model.pt / patchtst_scaler.pkl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import subprocess\n",
+    "result = subprocess.run(\n",
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)